File: oldproject/arouterdemo/src/main/java/li/glorin/arouterdemo/MainActivity.java
package li.glorin.arouterdemo;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import com.alibaba.android.arouter.launcher.ARouter;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onClick(View view) {
        switch (view.getId()) {
            case R.id.buttonTest:
                ARouter.getInstance().build("/test/activity")
                        .withString("date", "Today")
                        .navigation();
                break;
        }
    }
}

File: FrescoDemo/app/src/main/java/xyz/glorin/frescodemo/MainActivity.kt
package xyz.glorin.frescodemo

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.facebook.drawee.view.SimpleDraweeView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<SimpleDraweeView>(R.id.draweeView).setImageURI(IMAGE_URL)
    }

    companion object {
        private const val IMAGE_URL = "https://frescolib.org/static/logo.png"
    }
}